frontend/packages/ux-editor/src/components/Elements/types/ConfigPageType.ts

```diff
@@ -0,0 +1 @@
+export type ConfPageType = 'receipt' | 'payment' | 'subform';
```

Comment:
I agree with your point. I’ll take a closer look at that. I’ll also need to refactor a bit since the `Conf` part already exists in the codebase. I moved the type out to enhance code clarity.

Reply:
This will be handled in a new PR